src/sentry/db/models/manager.py

```diff
@@ -85,13 +88,13 @@ def __init__(self, *args, **kwargs):
         #: project slug is not.
         self.cache_fields = kwargs.pop("cache_fields", [])
         self.cache_ttl = kwargs.pop("cache_ttl", 60 * 5)
-        self._cache_version = kwargs.pop("cache_version", None)
+        self._cache_version: Optional[str] = kwargs.pop("cache_version", None)
         self.__local_cache = threading.local()
         super().__init__(*args, **kwargs)
 
     @staticmethod
     @contextmanager
-    def local_cache():
+    def local_cache() -> Any:
```

Comment:
actually since you're using the decorator you need to return `Generator` here, then the decorator will redefine this method into returning a context manager

it may not be useful to type this method since at the end mypy will correctly infer that it's a context manager